src/ExtensionsSample/Samples/DocumentDBSamples.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+
+namespace ExtensionsSample.Samples
+{
+    // To use the DocumentDB samples:
+    // 1. Create a new DocumentDB account
+    // 2. Add the DocumentDB Endpoint Uri to the 'AzureWebJobsDocumentDBUri' App Setting in app.config
+    // 3. Add the DocumentDB Primary Key to the 'AzureWebJobsDocumentDBKey' App Setting in app.config
+    // 4. Add typeof(DocumentDBSamples) to the SamplesTypeLocator in Program.cs
+    public static class DocumentDBSamples
+    {
+        // POCO Output binding
+        //   The binding inserts the newly created item into the specified Collection when the
+        //   method successfully exits.
+        //   If the database or the collection do not exist, they are automatically created.
+        //
+        // Other supported types:
+        //   out T[]
+        //   IAsyncCollector<T>
+        //   ICollector<T>
+        public static void InsertDocument(
+            [TimerTrigger("00:01")] TimerInfo timer,
+            [DocumentDB(DatabaseName = "ItemDb", CollectionName = "ItemCollection", CreateIfNotExists = true)] out Item newItem)
```

Comment:
Also, do you have any DbName defaulting in place, like you have for EasyTables? I wonder if the Document DB SDK has a declarative mapping attribute or anything people could apply to their poco types that we could read/use. That's just a future thought.